### DISCUSSION TITLE
Is prompt same as prefix?

### DISCUSSION THREAD

**Discussion by kouohhashi** [openai/whisper#1080] [Q&A] [answered]
I saw we  can use prompt on OpenAI audio API.
I says with prompt we can tweak inference results.
How can we use prompt on the open Whisper model?

Is it same as using prefix option at inference time?

**Comment by jongwook** [comment, marked as answer]
The `prompt` option in the API is equivalent to the `initial_prompt` in the `transcribe()` function in this python package. For the distinction between prompt and prefix, please see:

- https://github.com/openai/whisper/discussions/117#discussioncomment-3727051
